<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 로드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="procBtn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgSrc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pawn.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="imgDst">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../orange.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>851</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
